java/api/src/main/java/io/ray/api/id/ObjectId.java

```diff
@@ -42,4 +44,13 @@ public int size() {
     return LENGTH;
   }
 
+  /**
+   * Generate a nil ObjectId.
+   */
+  private static ObjectId genNil() {
```

Comment:
Nit: better avoid exposing nil ID in the Java API module. I think we can pass a null into `nativeRegisterObjectRef` instead, and convert null to nil ID in C++.

Reply:
updated